### DISCUSSION TITLE
Is Windows binaries are not supported yet?

### DISCUSSION THREAD

**Discussion by zackfall** [semantic-release-cargo/semantic-release-cargo#91] [Q&A]
I was watching the README file, and I saw that it only accepts linux and macOS binaries, I mean, I can do it for my self, yeah, but I don't know if it'll be a problem if I just use something is not supported yet

**Comment by EricCrosson** [comment]
I haven't tried running this on any Windows systems because I don't own any! I'm not sure what you might run into

**Comment by zackfall** [comment]
Hmm, okay, I will try it when I have time, if something goes wrong, I'll open an issue
